ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
re: glibc, I meant that I won't sanction an image based on uclibc or musl. As long as its based on glibc it's fine with me. Regarding pushing the actual image build on master, @junr03 what do you think?